presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/TestHive.java

```diff
@@ -80,4 +81,17 @@ public void testHideDeltaLakeTables()
 
         throw new SkipException("not supported");
     }
+
+    @Test
+    public void testHiveViewTranslationError()
+    {
+        try (Transaction transaction = newTransaction()) {
+            assertThatThrownBy(() -> transaction.getMetadata().getView(newSession(), view))
+                    .isInstanceOf(HiveViewNotSupportedException.class)
+                    .hasMessageContaining("Hive views are not supported");
+
+            // TODO: combine this with tests for successful translation in TestHiveViews product test, it is
+            // currently not possible because translation is not controlled by a session property
```

Comment:
>  currently not possible ...

this is not true, it is possible (https://github.com/prestosql/presto/pull/5636#issuecomment-715210463)
at most, we decided not to do this

```suggestion
            // TODO: combine this with tests for successful translation in TestHiveViews product test
```

Reply:
@findepi does this looks better? 

```
// TODO: combine this with tests for successful view translation (currently in TestHiveViews product test)
```

Reply:
nice, thanks